events/wc_lifestyle_events.txt

```diff
@@ -0,0 +1,304 @@
+			}
+		}
+		if = {
+			limit = { has_character_flag = brew_martial }
+			clr_character_flag = brew_martial
+			add_artifact = sulfuron_slammer_keg
+			new_artifact = {
+				set_name = "[This.GetName] 3/3"
+				set_artifact_flag = keg_fullness_3
+			}
+		}
+		if = {
+			limit = { has_character_flag = brew_stewardship }
+			clr_character_flag = brew_stewardship
+			add_artifact = metal_brew_keg
+			new_artifact = {
+				set_name = "[This.GetName] 3/3"
+				set_artifact_flag = keg_fullness_3
+			}
+		}
+		if = {
+			limit = { has_character_flag = brew_intrigue }
+			clr_character_flag = brew_intrigue
+			add_artifact = tiger_stout_keg
+			new_artifact = {
+				set_name = "[This.GetName] 3/3"
+				set_artifact_flag = keg_fullness_3
+			}
+		}
+		if = {
+			limit = { has_character_flag = brew_learning }
+			clr_character_flag = brew_learning
+			add_artifact = serpent_ale_keg
+			new_artifact = {
+				set_name = "[This.GetName] 3/3"
+				set_artifact_flag = keg_fullness_3
+			}
+		}
+		clr_character_flag = do_not_disturb
+	}
+}
+character_event = {
+	id = WCLIFES.30
+	picture = GFX_evt_brewmaster
+	desc = EVTDESC_WCLIFES.30
+	border = GFX_event_normal_frame_diplomacy
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = EVTOPTA_WCLIFES.30
+		trigger = { has_artifact = halaani_whiskey_keg }
+		event_target:brew_receiver = {
+			set_character_flag = drank_brew_diplomacy
+			character_event = { id = WCLIFES.35 }
+		}
+	}
+	option = {
+		name = EVTOPTB_WCLIFES.30
+		trigger = { has_artifact = sulfuron_slammer_keg }
+		event_target:brew_receiver = {
+			set_character_flag = drank_brew_martial
+			character_event = { id = WCLIFES.35 }
+		}
+	}
+	option = {
+		name = EVTOPTC_WCLIFES.30
+		trigger = { has_artifact = metal_brew_keg }
+		event_target:brew_receiver = {
+			set_character_flag = drank_brew_stewardship
+			character_event = { id = WCLIFES.35 }
+		}
+	}
+	option = {
+		name = EVTOPTD_WCLIFES.30
+		trigger = {
+			OR = {
+				has_artifact = tiger_stout_keg
+				has_artifact = serpent_ale_keg
+			}
+		}
+		character_event = { id = WCLIFES.31 }
+	}
+}
+character_event = {
+	id = WCLIFES.31
+	picture = GFX_evt_brewmaster
+	desc = EVTDESC_WCLIFES.30
+	border = GFX_event_normal_frame_diplomacy
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = EVTOPTA_WCLIFES.31
+		trigger = { has_artifact = tiger_stout_keg }
+		event_target:brew_receiver = {
+			set_character_flag = drank_brew_intrigue
+			character_event = { id = WCLIFES.35 }
+		}
+	}
+	option = {
+		name = EVTOPTB_WCLIFES.31
+		trigger = { has_artifact = serpent_ale_keg }
+		event_target:brew_receiver = {
+			set_character_flag = drank_brew_learning
+			character_event = { id = WCLIFES.35 }
+		}
+	}
+	option = {
+		name = EVTOPTD_WCLIFES.30
+		trigger = {
+			OR = {
+				has_artifact = halaani_whiskey_keg
+				has_artifact = sulfuron_slammer_keg
+				has_artifact = metal_brew_keg
+			}
+		}
+		character_event = { id = WCLIFES.30 }
+	}
+}
+character_event = {
+	id = WCLIFES.35
```

Comment:
Don't affect your relations with character.
Also, you can't refuse to drink it.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/91624164-5a2e1080-e9a7-11ea-8f29-8d3a30aea66b.png)

</details>

Reply:
You can now refuse the drink